src/mantine-core/src/components/Image/BackgroundImage/BackgroundImage.tsx

```diff
@@ -11,6 +11,7 @@ import { Box } from '../../Box';
 interface _BackgroundImageProps extends DefaultProps, React.ComponentPropsWithoutRef<'div'> {
   src: string;
   radius?: MantineNumberSize;
+  padding?: MantineNumberSize;
```

Comment:
Should not be here, remove it

Reply:
I removed the prop, sorry I was getting ts errors for some reason.